### PR TITLE
update command description in CLI

### PR DESCRIPTION
**Pull request by allencloud**
**\- What I did**
1. correct some wrong description of docker commandline
2. change Args Number of a command, such as command plugin
3. update related docs

**\- How I did it**

**\- How to verify it**

**\- Description for the changelog**

**\- A picture of a cute animal (not mandatory but encouraged)**

Signed-off-by: allencloud allen.sun@daocloud.io

### CODE REVIEW

**Comment by SvenDowideit** [comment]
test this pleae

**Comment by SvenDowideit** [comment]
test this please

**Comment by SvenDowideit** [comment]
I'm trying to re-trigger the docs checker

File: docs/reference/commandline/volume_rm.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ parent = "smn_cli"
 # volume rm
 
 ```markdown
-Usage:  docker volume rm VOLUME [VOLUME]...
+Usage:  docker volume rm VOLUME [VOLUME...]
 
-Remove a volume
+Remove one or more volume
```
Comment (adamtheturtle) [inline docs/reference/commandline/volume_rm.md]: This should be "Remove one or more volumes".

**Comment by thaJeztah** [comment]
LGTM after https://github.com/docker/docker/pull/25240#discussion_r72874032 has been addressed

File: docs/reference/commandline/volume_rm.md (hunk 1 above)
Comment (allencloud) [inline docs/reference/commandline/volume_rm.md]: Thanks. Great. Updated

**Comment by vdemeester** [comment]
LGTM 👼

**Comment by SvenDowideit** [comment]
this is not really just docs - its got code file changes.

**Comment by thaJeztah** [comment]
@SvenDowideit not strictly needed, please read https://github.com/docker/docker/blob/master/project/PATCH-RELEASES.md

**Pull request merged**